### ISSUE TITLE
Data in the hover over on resource bar elements in the list of resources viewed can't be navigated to using keyboard

### ISSUE THREAD

**Issue by jennlove-um** [tl-its-umich-edu/my-learning-analytics#1190]
Users should be able to navigate to or access all content in the Resources Accessed by using the keyboard.

The bars on the Resources Accessed bar chart include metadata about file views in a hover-over.  Tabbing should allow users to access the information that is in the hover-over data.

**Comment by ssciolla** [comment]
@jennlove-um, please let me know if you need guidance on testing my changes. There's a video in the linked PR that is probably a good starting point. #1472
